setup.py

```diff
@@ -27,7 +27,7 @@
     ],
     keywords="bonsai",
     install_requires=[
-        "gym==0.9.7",
+        "gym>=0.9.7",
```

Comment:
Same as the bonsai-common changes, it's safer to lockdown the version.

Reply:
I agree it is safer for mba. For gym, however, many users may be using a newer version. Do we want to continue using 0.9.7 or 0.18.0 or a >= 0.9.7

Reply:
pinning to 0.18.0!